js/app.js

```diff
@@ -0,0 +1,91 @@
+console.log('sanity check');
+let xhr = new XMLHttpRequest();
```

Comment:
This should be refactored so that you're using the shortXhr you defined below here.